en/settings.md

```diff
@@ -9,11 +9,14 @@ Standard YFM settings are listed below. Depending on the tool you use, you can c
 | Name | Description | Type | Default value |
 | :--- | :--- | :--- | :--- |
 | `vars` | [Variables](./syntax/vars.md) | `Object` | `{}` |
-| `allowHTML` | Allow HTML | `bool` | `false` |
+| `allowHTML` | Allow HTML in markup | `bool` | `false` |
 | `linkify` | Convert link-like strings to links | `bool` | `false` |
 | `breaks` | Use the carriage return character for a line break | `bool` | `true` |
 | `conditionsInCode` | Meet conditions in code blocks | `bool` | `false` |
 | `disableLiquid` | Disable the use of variables | `bool` | `false` |
 | `supportGithubAnchors` | Generate additional [anchors](./syntax/base.md#headers) that are compatible with GitHub | `bool` | `false` |
 | `lang` | Localization language of default texts | `string` | `ru` |
+`needFlatListHeadings` | Generate a flat list of all headers in a document | `bool` | `false`

```

Comment:
Bad mix. This is for common settings for yfm-transform and yfm-docs. `needFlatListHeadings` isn't actually for yfm-docs. Need move to yfm-transform settings: https://ydocs.tech/ru/tools/transform/settings